test/intervals/TestIntervals.jl

```diff
@@ -183,7 +181,9 @@ facts("IntervalCollection") do
 
 
     context("Show") do
-        nullout = open("/dev/null", "w")
+        nullstream = @windows ? "NUL" : "/dev/null"
+        nullout = open(nullstream, "w")
```

Comment:
```
julia> write(DevNull, "Hi")
2

```

Do you use Julia v0.4.2? It works on my computer. The behavior you saw is a bug I fixed :)